firmware_tools/ghidra/vxhunter_core.py

```diff
@@ -268,54 +265,45 @@ def get_symbol_table(self):
             else:
                 unpack_format = '<I'
             symbol_name_addr = int(struct.unpack(unpack_format, symbol_name_addr)[0])
-            self.logger.debug("symbol_name_addr: %s" % symbol_name_addr)
             symbol_dest_addr = int(struct.unpack(unpack_format, symbol_dest_addr)[0])
-            self.logger.debug("symbol_dest_addr: %s" % symbol_dest_addr)
-            self._symbol_table.append({
-                'symbol_name_addr': symbol_name_addr,
-                'symbol_name_length': None,
-                'symbol_dest_addr': symbol_dest_addr,
-                'symbol_flag': symbol_flag,
-                'offset': i
-            })
+            self.logger.debug("symbol_name_addr: {}; symbol_dest_addr: {}".format(symbol_name_addr, symbol_dest_addr))
+            self._symbol_table.append({'symbol_name_addr': symbol_name_addr, 'symbol_name_length': None, 'symbol_dest_addr': symbol_dest_addr, 'symbol_flag': symbol_flag, 'offset': i})
         # self.logger.debug("self._symbol_table: %s" % self._symbol_table)
-        self.logger.debug("len(self._symbol_table): %s".format(len(self._symbol_table)))
+        self.logger.debug("len(self._symbol_table): {}".format(len(self._symbol_table)))
         self._symbol_table = sorted(self._symbol_table, key=lambda x: x['symbol_name_addr'])
         for i in range(len(self._symbol_table) - 1):
             self._symbol_table[i]['symbol_name_length'] = self._symbol_table[i + 1]['symbol_name_addr'] - \
-                                                        self._symbol_table[i]['symbol_name_addr']
-        self.logger.debug("len(self._symbol_table): %s".format(len(self._symbol_table)))
+                                                          self._symbol_table[i]['symbol_name_addr']
+        self.logger.debug("len(self._symbol_table): {}".format(len(self._symbol_table)))
         return True
 
     @staticmethod
     def _is_printable(c):
-        """ Check Char is printable.
+        """
+        Check Char is printable.
 
         :param c: char to check.
         :return: True if char is printable, False otherwise.
         """
-        return 32 <= ord(c) <= 126
+        try:
```

Comment:
Using decode('ascii') will contain some unprintable ascii character like \x00, might be we should keep using 32 <= ord(c) <= 126 for now.